pcdsdevices/epics_motor.py

```diff
@@ -125,17 +127,43 @@ def limits(self, limits):
         self.low_limit = limits[0]
         self.high_limit = limits[1]
 
+    def enable(self, enable):
```

Comment:
I expect a user will see a method named `enable` and immediately expect that calling `motor.enable()` will enable a motor. Perhaps we can be kind to the user and default this to `enable=True`. If you don't expect most users to try to call this you can ignore this comment.